tests/modelfree/test_experiments.py

```diff
@@ -53,6 +59,16 @@ def test_score_agent(config):
     outcomes = [run.result[k] for k in ['ties', 'win0', 'win1']]
     assert sum(outcomes) == run.config['episodes']
 
+    if config.get('record_traj', False):
+        try:
+            for i in range(2):
+                traj_file_path = os.path.join(config['record_traj_params']['save_dir'],
+                                              f'agent_{i}.npz')
+                assert os.path.exists(traj_file_path)
```

Comment:
Don't feel this super strongly, but do somewhat believe that it could be valuable to have this test actually make some assertions about the structure of the numpy object that's being saved, to confirm it matches expectations

Reply:
I've added some basic additional tests here.